ozaria/site/views/play/level/tome/SpellPaletteView.coffee

```diff
@@ -206,7 +206,7 @@ module.exports = class SpellPaletteView extends CocoView
     @$el.toggleClass 'controls-disabled', not enabled
 
   onFrameChanged: (e) ->
-    return unless e.selectedThang?.id is @thang.id
+    return unless e.selectedThang?.id is @thang?.id
```

Comment:
Reason for change?

Reply:
For some level, it was giving an error because thang was not defined, hence added this check